src/MICore/UnixUtilities.cs

```diff
@@ -42,6 +42,12 @@ public static class UnixUtilities
             string pidFifo,
             string debuggerCmd)
         {
+            // On OSX, 'wait' will return once there is a status change from the launched process rather than for it to exit, so
+            // we need to use 'fg' there. 
+            // On Linux, fg will fail with 'no job control' because our bash shell is not interactive, so we need 
```

Comment:
I can play around with it on Tuesday. I am not sure if it is a difference in the way bash gets started (AppleScript vs. Linux launches), or if it is just because OSX uses a different implementation/version of bash.